Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -263,6 +263,9 @@ def __len__(self):
 flagReserved1 = 0x40
 flagReserved2 = 0x80
 
+# These flags are kept for XML output after decompiling the coordinates
+keepFlags = flagOnCurve + flagReserved1
```

Comment:
Shouldn’t `flagReserved1` be renamed now? It is no longer a reserved flag in the current version of the spec (`OVERLAP_SIMPLE` in https://docs.microsoft.com/en-us/typography/opentype/spec/glyf).

Reply:
good point